src/backend/distributed/commands/multi_copy.c

```diff
@@ -93,8 +94,8 @@ static void OpenCopyConnections(CopyStmt *copyStatement,
 								ShardConnections *shardConnections, bool stopOnFailure,
 								bool useBinaryCopyFormat);
 
-static bool CanUseBinaryCopyFormat(TupleDesc tupleDescription,
-								   CopyOutState rowOutputState);
+static bool CanUseBinaryCopyFormat(TupleDesc tupleDescription);
+static bool BinaryOutputFunctionIsDefined(Oid typeId);
```

Comment:
Just remove the "Is" from the function name. We generally use question form and drop the is, has,can etc. (CanUseBinaryCopyFormat is an exception, because otherwise it becomes difficult to understand what it does)